common_clang.cpp

```diff
@@ -87,7 +87,12 @@ static llvm::sys::Mutex lazyCCInitMutex;
 
 static llvm::ManagedStatic<llvm::sys::SmartMutex<true> > compileMutex;
 
-void CommonClangTerminate() { llvm::llvm_shutdown(); }
+void CommonClangTerminate() {
+  llvm::llvm_shutdown();
+#ifndef USE_PREBUILT_LLVM
```

Comment:
why should we add this macro here?

Reply:
deleteManagedStaticMutex doesn't exist in pre-built llvm.
It only exists when we use llvm source and apply 0001-Adding-llvm-deleteManagedStaticMutex.patch to llvm.